src/datasets_labeled.py

```diff
@@ -644,4 +644,25 @@ def clean_ceo_data(df: pd.DataFrame) -> pd.DataFrame:
             ),
         ),
     ),
+    LabeledDataset(
+        dataset="Malawi_corrected",
+        country="Malawi",
+        processors=(
+            Processor(
+                filename="Crops.shp",
+                crop_prob=1.0,
+                start_year=2021,
+            ),
+            Processor(
+                filename="Noncrops.shp",
+                crop_prob=0.0,
+                start_year=2021,
+            ),
+            Processor(
+                filename="Major_protected_areas.shp",
+                crop_prob=0.0,
+                start_year=2021,
```

Comment:
Is there a particular reason for using 2021 here?
Malawi corrected was made using the Malawi 2020 map so the `start_year` should be 2020. Unfortunately, it looks like the tifs will need to be reexported for this. Apologies if this was not super clear

Reply:
My oops, I meant to double check which year to list before pushing and forgot. I went off initial email from Brian that said 2021, but reexporting tifs won't be a problem! I'll go ahead and close this pr.